auth/realm/base/src/main/java/org/wildfly/security/auth/realm/ElytronMessages.java

```diff
@@ -146,4 +146,11 @@
     @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 13004, value = "JAAS logout failed for principal %s")
     void debugInfoJaasLogoutFailure(Principal principal, @Cause Throwable cause);
+
+    @Message(id = 13005, value = "Signature for the following identity is invalid: %s.")
```

Comment:
Very minor comment, the trailing period can be removed here and below.

Reply:
Removed